File: CodeEvalSolutionsJava/OddNumber/src/OddNumbers.java
/**
 * Created by kenny on 5/20/17.
 */
public class OddNumbers {

    public static void main(String[] args){

        print_odd_numbers(1, 99);
    }


    public static void print_odd_numbers(int start, int end){
        for(int i=start; i<=end; i++){
            if(i%2 != 0){
                System.out.println(i);
            }
        }
    }
}

File: CodeEvalSolutionsPHP/OddNumbers.php
<?php

function print_odd_numbers($start, $end){
    
    for($i=$start; $i<=$end; $i++){
        if($i%2 != 0){
            echo $i . PHP_EOL;
        }
    }
}

print_odd_numbers(1, 99);

?>
